packages/create-gatsby/src/install-plugins.ts

```diff
@@ -0,0 +1,26 @@
+import { reporter } from "./reporter"
+
+export async function installPlugins(
+  plugins: Array<string>,
+  pluginOptions: Record<string, Record<string, any> | undefined> = {},
+  rootPath: string
+): Promise<void> {
+  let installPluginCommand
+
+  try {
+    installPluginCommand = require.resolve(`gatsby-cli/lib/plugin-add`, {
+      paths: [rootPath],
+    })
+  } catch (e) {
+    // The file is missing
+  }
+
+  if (!installPluginCommand) {
+    reporter.error(`gatsby-cli not installed, or is too old`)
+    return void 0
```

Comment:
Nitpick: I don't think there's a reason to use the (less common) `void 0` form of `undefined` in modern JavaScript 😅 

```suggestion
    return undefined
```

Reply:
It's because our linting rules don't let me just do `return` if I'm returning a value in another code path (which I was before). They are different types, so I can't return `undefined` if it's a void return-typed function. I've changed them to both do `return`

Reply:
🤦 CLASSIC LINTING